assets/js/blocks/cart-checkout/checkout/edit.js

```diff
@@ -268,6 +271,11 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
 
 const CheckoutEditor = ( { attributes, setAttributes } ) => {
 	const { className } = attributes;
+
+	if ( attributes?.isPreview ) {
```

Comment:
you can destruct `isPreview` from attributes as well, it will return undefined if it's not set.